doc/acting/discontinued/teb/motion_planner.py

```
@@ -0,0 +1,1028 @@
+        return speed
+
+    def __get_speed_overtake(self, behavior: str) -> float:
+        speed = 0.0
+        if behavior == bs.ot_app_blocked.name:
+            speed = self.__calc_speed_to_stop_overtake()
+        elif behavior == bs.ot_app_free.name:
+            speed = self.__calc_speed_to_stop_overtake()
+        elif behavior == bs.ot_wait_stopped.name:
+            speed = bs.ot_wait_stopped.speed
+        elif behavior == bs.ot_wait_free.name:
+            speed == self.__get_speed_cruise()
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)